resty

```diff
@@ -53,6 +53,7 @@ HELP
         _RESTY_DATA_DIR="$_RESTY_CONF_DIR"
     fi
     mkdir -p "$_RESTY_CONF_DIR"
+    mkdir -p "$_RESTY_CONF_DIR/c"
```

Comment:
The `-p` is redundant or you can remove the mkdir call above. What you have now is the safest though.